package.json

```diff
@@ -2,14 +2,33 @@
   "name": "@custom-elements-manifest/root",
   "private": true,
   "license": "MIT",
+  "type": "module",
   "repository": {
     "type": "git",
     "url": "https://github.com/open-wc/custom-elements-manifest.git"
   },
-  "scripts": {},
+  "scripts": {
+    "test": "yarn workspaces run test"
+  },
   "dependencies": {},
-  "devDependencies": {},
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^19.0.0",
+    "@rollup/plugin-node-resolve": "^13.0.0",
+    "@rollup/plugin-replace": "^2.4.2",
+    "@small-tech/esm-tape-runner": "^1.0.3",
+    "custom-elements-manifest": "^1.0.0",
+    "esbuild": "^0.12.15",
+    "nodemon": "^2.0.4",
+    "npm-run-all": "^4.1.5",
+    "rollup": "^2.52.8",
+    "rollup-plugin-terser": "^7.0.2",
+    "tap-spec": "^5.0.0",
+    "tape": "^5.2.2",
```

Comment:
Could we align test runners/frameworks for all the packages? it feels a bit wasteful to have different ways of testing for different package

Reply:
yeah np